microsoft-edge/devtools-guide-chromium/memory-problems/memory-101.md

```diff
@@ -23,216 +23,166 @@ ms.date: 07/21/2023
-## Cycles
-
-_Cycles_ are nodes that appear at least twice in a retainer path.
-One appearance of a node is earlier in the retainer path, and other appearances of that node are later in the retainer path.
+*  Externally in the **renderer's memory**. A _wrapper object_ is created and used for accessing external storage where, for example, script sources and other content that's received from the Web is stored, rather than copied onto the VM heap.
 
-To free up memory, it's most important to remove the occurrence of the node which appears first in the retainer path.
-The second and potentially subsequent appearances of the node are still displayed in the **Retainers** section.
 
+<!-- ------------------------------ -->
+#### JavaScript objects
 
-### Using filters to hide cycles
+Memory for new JavaScript objects is allocated from a dedicated JavaScript heap (or _VM heap_). These objects are managed by V8 VM's garbage collector, and therefore, these objects stay alive as long as there is at least one reference to them.
 
-Cycles are displayed in the **Retainers** section of a heap snapshot.
-To help simplify the retainer path, the **Retainers** section in the **Memory** tool has filters to hide cycles.
 
-In the **Retainers** section, a cycled node is indicated by being grayed out.
+<!-- ------------------------------ -->
+#### Other objects
 
-In the following image, in the **Filter edges** dropdown menu, **Hide cycled** is not selected, so a cycled node (grayed out) is displayed:
+* **Native objects**: anything not in the JavaScript heap is called a _native object_. A native object, in contrast to a heap object, isn't managed by the V8 garbage collector throughout its lifetime, and can only be accessed from JavaScript by using its JavaScript wrapper object.
 
-![In the 'Filter edges' dropdown menu, 'Hide cycled' is not selected](memory-101-images/filters-retainers-memory-tool-no-hide-cycled.png)
+* **Concatenated strings**: strings that are stored and then joined together, as a result of a string concatenation in JavaScript, are stored as _concatenated strings_ in V8. The joining of the string contents occurs only as needed, such as when a substring of a joined string needs to be constructed.
 
-In the **Filter edges** dropdown menu, **Hide cycled** is selected, so the cycled node is not displayed:
+  For example, if you concatenate `a` and `b`, you get a concatenated string `(a, b)` which represents the result of concatenation. If you later concatenate `c` with that result, you get another concatenated string: `((a, b, c)`.
 
-![In the 'Filter edges' dropdown menu, 'Hide cycled' is selected](memory-101-images/filters-retainers-memory-tool-hide-cycled.png)
+* **Arrays**: an _array_ is an object that has numeric keys. Arrays are used extensively in the V8 VM for storing large amounts of data. Sets of key-value pairs that are used like dictionaries are implemented as **arrays**.
```

Comment:
```suggestion
* **Arrays**: an _array_ is an object that has numeric keys. Arrays are used extensively in the V8 VM for storing large amounts of data. Sets of key-value pairs that are used like dictionaries are implemented as arrays.
```
is bolding appropriate?  'arrays' seems like a plain regular word, after the introduction highlights it.  Same w/ next sentence.  If "arrays" is a lang keyword/type, it should be backticks, not bold.

Reply:
removed bold 2x, seems just as clear.